splinterd/api/static/openapi.yml

```diff
@@ -327,6 +327,145 @@ paths:
         500:
           description: Internal service error
 
+  /user/notifications:
+    get:
+      description: Retrieve a list of all notifications
+      responses:
+        200:
+          description: Notifications have been retrieved
+          content:
+            application/json:
+              schema:
+                type: array
+                $ref: '#/components/schemas/Notification'
+        403:
+          description: User not authorized to access this endpoint
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
+      description: Send a list of notifications to the user module
+      responses:
+        200:
+          description: Notifications have been published
+        400:
+          description: Malformed notifications list
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal service error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /user/users/{user}/notifications:
```

Comment:
Why `/user/users`?  `/user/{user}/notifications` seems a little cleaner to me

Reply:
/user -> user module
.../users -> users resource

I agree /users/{user}/... is cleaner but it doesn't follow the <module>/<resource>/<id> pattern  previously established in this file.

Reply:
Could we call it userModule instead? Or something...

Reply:
/accounts/users...?
/users/accounts...?